src/omit.ts

```diff
@@ -21,6 +21,8 @@ export function omit<T, K extends keyof T>(
     delete newRecordFields[k]
   })
 
-  // TODO: keep 'sloppyness'
-  return record(newRecordFields) as Runtype<any>
+  return internalRecord(
+    newRecordFields,
+    isNonStrictRuntype(original),
+  ) as Runtype<any>
```

Comment:
here and in similar runtypes: we check if the provided runtype is non-strict; if it is, the runtype returned by `omit` will also be non-strict, if not, `omit` will return the same.

`isNonStrictRuntype` just checks if the relevant symbol property is true